modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -5,8 +5,9 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with the application and the components
```

Comment:
This is not part of this PR, but please change the title to 'Interacting with applications and components' since this topic is about generic applications and not one particular application.

Reply:
Done